src/lib/types.ts

```diff
@@ -0,0 +1,51 @@
+import type { JwtPayload } from 'jsonwebtoken'
+
+export interface VerifyAppleIdTokenParams {
+  idToken: string
+  clientId: string | string[]
+  nonce?: string
+}
+
+export type PJwtPayload = Pick<
```

Comment:
From my point of view is much better to use the whole `JwtPayload` type instead of the `Pick` type because we dont need to limit our users just with these fields. They can use all the fields which `JwtPayload` supports.

Or is there any other reason why you picked just some of the fields?

Reply:
removed